data/languages/en_US.cfg

```diff
@@ -2,4 +2,5 @@
     name="English (US)"
     locale=en_US
     alternates=C, en_PH
+    percent=100
```

Comment:
Why not make this key optional?

Reply:
Because this is literally the _only_ file that does not include it. It doesn't make sense to me to make it optional just so one file can omit it, when it makes just as much sense for that one file to set it to 100.